ruby/cross_service_examples/item-tracker/src/db_wrapper.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+require "logger"
+require "sequel"
+require "multi_json"
+require_relative "report"
+
+class RDSResourceError < Exception
+end
+
+class RDSClientError < Exception
+end
+
+# Issues commands directly to the Amazon RDS Data Service, including SQL statements.
+class DBWrapper
+
+  # @param config [List]
+  # @param rds_client [AWS::RDS::Client] An Amazon RDS Data Service client.
+  def initialize(config, rds_client)
+    @cluster = config["resource_arn"]
+    @secret = config["secret_arn"]
+    @db_name = config["database"]
+    @table_name = config["table_name"]
+    @rds_client = rds_client
+    @model = Sequel::Database.new
+    @logger = Logger.new($stdout)
+  end
+
+  # Helper method to prep SQL for execution
+  # @param sql [String]
+  # @return [String]
+  def _format_sql(sql)
```

Comment:
Underscore is a python idiom. If this is truly private, you can put `private` in the class and everything defined under is private.
```
class Foo
  def public; end

  private

  def private; end
end
```

Reply:
Done.